test/server/routes/__mocks__/borrowData.js

```diff
@@ -0,0 +1,23 @@
+export const newBookToBorrow = {
```

Comment:
You should transfer the objects in this file to the bookData.js file

Reply:
Noted, done